editor/create_dialog.cpp

```diff
@@ -103,25 +103,35 @@ void CreateDialog::popup_create(bool p_dont_clear, bool p_replace_mode) {
 		popup_centered(popup_size);
 	}
 
-	if (p_dont_clear) {
-		search_box->select_all();
-	} else {
-		search_box->clear();
-	}
+	is_replace_mode = p_replace_mode;
 
-	search_box->grab_focus();
+	if (p_replace_mode) {
+		
```

Comment:
Travis reports two extraneous tabs on this line, though I can't see them on github.